src/asInput/index.jsx

```diff
@@ -181,6 +181,11 @@ const asInput = (WrappedComponent, inputType = undefined, labelFirst = true) =>
       );
     }
 
+    generateInputGroupAddonKey({ prefix, index }) {
+      // Prefix is either prepend or append
```

Comment:
Why is this comment relevant? 

What I mean is, why should an implementer of this method care what the prefix has been used for? 

It seems like if this comment should live anywhere it should live where this method is actually getting implemented, where `prefix` is probably `prepend` or `append` in that implementation context.